### DISCUSSION TITLE
11/12(火) 20:00~ 全体ミーティング　#3

### DISCUSSION THREAD

**Discussion by ayase-mstk** [42-pong/42-pong#15] [Meetings]
議題
- 進捗共有
- 次回タスク

**Comment by penguin2** [comment]
以下、軽く話し合いできればと思います。
- Issue や PR のラベル
- default branch
- PR レビュワールール
- coderabbit: 設定、使い方
-

**Comment by habvi** [comment]
BE 進捗
- 全体に関すること
  - [x] コンテナ起動, Makefile 作成
  - [x] 各種チームでのルール作成 (PR, commit, branch) 
  - [x] openapi.yaml 作成
  - [x] coderabbit 導入
  - [x] PR template 作成
- BE に関すること
  - Django で実装していくにあたって、環境整備
    - [x] Python の linter, formatter, 型check
    - [x] Django 初期設定
    - [x] swagger-ui
    - [ ] DB コンテナをたて、Djnago の設定を postgres に変更
  - [x] `/health` GET の簡単な実装 (actions で unit-test も走る)

話したいこと
- `/health` GET の次の機能実装内容について
- workflow 
  - unit test を走らせる対象をどこに指定するか
- PR
  - 最後に approve した人が merge するか、PR 作成者が merge するか (回答が終わってないかもしれないので PR 作成者？)
  - merge 先が dev ではない PR の場合、merge 先が merge されるまでレビューに出すのを待つ？
  - 短い期間だけのルールになりそうですが、coderabbit の review 修正を終えてから review 依頼出す方が良さそう
  - `FYI`, `TIPS` みたいな補足情報・リアクションはなくても良い、だけのも追加したい
- project
  - 数字も付けられるけどいるかどうか (task の重さ想定に使えそう)

**Comment by habvi** [comment]
## 議事録
- PR の Label
  - 自由に追加する。ゆるく PR につける
- PR reviewer 方法
  - チーム外の人を 1 人ランダムで指定し、その人が自分のチームに後で共有する
- workflow
  - unit test など範囲が明らかなものは `paths` で指定する
- PR
  - PR 作成者が merge する
  - dev への merge じゃなくても review 依頼出して良い、けど branch の依存は dev から 2 層目まで
- project
  - 数字入力ができる category を作ったが必須ではない。書く場合、かかった時間を数字で。
